ui/app/services/flags.ts

```diff
@@ -31,15 +31,15 @@ export default class flagsService extends Service {
     this.flags = flags;
   }
 
-  get managedNamespaceRoot() {
-    if (this.flags && this.flags.includes(FLAGS.vaultCloudNamespace)) {
-      return 'admin';
-    }
-    return null;
+  get isManaged(): boolean {
+    return this.flags && this.flags.includes(FLAGS.vaultCloudNamespace);
```

Comment:
`this.flags` will always be truthy because the default value is an empty array, right? So I think we can remove the first part of the conditional here. 🤔  If we're concerned that the value passed to `setFeatureFlags` is undefined then I wonder if a more long-term solution is instead adding a conditional check above so that we can rely on `this.flags` always being an array☝️
```js
 setFeatureFlags(flags: string[]) {
    this.flags = flags ? flags : [];
  }

Reply:
Good catch. However, I did some additional moving to make the pattern more consistent and clear in the service how we were setting the tracked property flags (which I now call `featureFlags`). You'll see I moved the one await fetch call from the application route to this service. Curious if this reads better.